### ISSUE TITLE
Sometimes it doesn't schedule and sometimes it does.

### ISSUE THREAD

**Issue by player99963** [ArtVentureX/sd-webui-agent-scheduler#218]
I've experienced this many times.
I thought it was a conflict with an extension, but often after closing an extension, even if it works fine at first, after restarting or refreshing the page, it suddenly fails to schedule.
Also, since it doesn't generate any messages in the console, I have no way of knowing what's going on.

![image](https://github.com/ArtVentureX/sd-webui-agent-scheduler/assets/36121614/7133887d-b1dd-4dbc-9380-a3013b99b493)
![image](https://github.com/ArtVentureX/sd-webui-agent-scheduler/assets/36121614/e9ec95d9-e304-4b21-906f-af9c2b7cbb41)

`venv "D:\sd-webui-aki-v4.6.1\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: v1.7.0
Commit hash: cf2772fab0af5573da775e7437e6acdca424f26e
Launching Web UI with arguments: --theme dark --api
no module 'xformers'. Processing without...
no module 'xformers'. Processing without...
No module 'xformers'. Proceeding without it.
Civitai Helper: Get Custom Model Folder
Tag Autocomplete: Could not locate model-keyword extension, Lora trigger word completion will be limited to those added through the extra networks menu.
Using sqlite file: D:\sd-webui-aki-v4.6.1\extensions\sd-webui-agent-scheduler\task_scheduler.sqlite3
sd-webui-prompt-all-in-one background API service started successfully.
Loading weights [07792c9874] from D:\sd-webui-aki-v4.6.1\models\Stable-diffusion\Pony Diffusion V6\nothingSDXL_nothingSDXLV067.safetensors
Civitai Helper: Settings:
Civitai Helper: max_size_preview: True
Civitai Helper: skip_nsfw_preview: False
Civitai Helper: open_url_with_js: True
Civitai Helper: proxy:
Civitai Helper: use civitai api key: False
Running on local URL:  http://127.0.0.1:7860
Creating model from config: D:\sd-webui-aki-v4.6.1\repositories\generative-models\configs\inference\sd_xl_base.yaml

To create a public link, set `share=True` in `launch()`.
Startup time: 12.0s (prepare environment: 1.4s, import torch: 1.9s, import gradio: 0.5s, setup paths: 0.5s, initialize shared: 0.2s, other imports: 0.3s, load scripts: 0.8s, create ui: 0.4s, gradio launch: 2.5s, app_started_callback: 3.3s).
Applying attention optimization: sdp-no-mem... done.
Model loaded in 7.1s (load weights from disk: 0.8s, create model: 0.2s, apply weights to model: 5.6s, calculate empty prompt: 0.2s).`
